x-pack/test/encrypted_saved_objects_api_integration/fixtures/es_archiver/encrypted_saved_objects/data.json

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
__question:__ do we need all these objects here? Can we just have `config`, `space` and `saved-object-with-migration` or even only `saved-object-with-migration`?

Reply:
I don't know 😬 
I just used what was dumped by esarchiver, I'll take a look 🤷

Reply:
Oh, oops, I misread your correction 🤦